src/components/users/TickForm.tsx

```diff
@@ -177,15 +235,23 @@ export default function TickForm ({ open, setOpen, setTicks, ticks, isTicked, cl
                       className='py-2 px-3 border border-gray-300 shadow-sm focus:ring-indigo-500 focus:border-indigo-500 block w-full sm:text-sm border-gray-300 rounded-md'
                     />
                   </div>
-                  <ComboBox options={styles} value={style} onChange={setStyle} label='Style' />
-                  <ComboBox options={attemptTypes} value={attemptType} onChange={setAttemptType} label='Attempt Type' />
+                  <ComboBox options={styles} value={style} onChange={handleStyleChange} label='Style' />
+                  <div className='flex items-center'>
+                    <label htmlFor='attemptType' className='block text-sm font-medium text-gray-700'>
+                      Attempt Type
+                    </label>
+                    <a href={climbingGlossaryLink} target='_blank' rel='noopener noreferrer' className='ml-2 text-gray-500 hover:text-gray-700' title='climbing terminology'>
```

Comment:
The little info button is a link to Climbing Glossary. 
![Screenshot 2025-01-25 at 6 33 55 PM](https://github.com/user-attachments/assets/15a03e50-39e3-40f9-a88e-9ece1f2471da)

Reply:
We have a Tooltip component you can use.  

Code https://github.com/OpenBeta/open-tacos/blob/develop/src/components/edit/form/AreaDesignationRadioGroup.tsx#L39-L41

---
<img width="922" alt="Screenshot 2025-01-26 at 12 31 07 PM" src="https://github.com/user-attachments/assets/0391db2e-f93f-4f76-b782-9cc9b7f5b21b" />